custom/icds_reports/views.py

```diff
@@ -2092,8 +2092,9 @@ def post(self, request, *args, **kwargs):
                     except InvalidLocationTypeException as e:
                         return JsonResponse({"message": e})
                     with open(export_file, 'r') as csv_file:
-                        icds_file, new = IcdsFile.objects.get_or_create(blob_id=blob_id, data_type=f'mbt_{data_type}')
+                        blob_id = f'{data_type}-{location_id}-{selected_date}'
```

Comment:
may be you can centralize this somewhere. Looks used a couple of place

Reply:
yeah, i can address this in follow up. I think that is true (and also already mostly centralized in the `get_cas_blob_file` method